inference-engine/src/vpu/common/src/ngraph/transformations/dynamic_to_static_shape_binary_elementwise.cpp

```diff
@@ -51,4 +55,14 @@ void dynamicToStaticShapeBinaryEltwise(std::shared_ptr<ngraph::Node> eltwise) {
     ngraph::replace_node(std::move(eltwise), std::move(outDSR));
 }
 
+} // namespace
+
+void dynamicToStaticShapeBinaryEltwise(std::shared_ptr<ngraph::Node> eltwise) {
+    if (eltwise->get_type_info() == ngraph::opset5::Select::type_info) {
+        processBinaryEltwise(eltwise, 1, 2);
```

Comment:
Seems a bit strange that `Select` ignores first input. If you assume it's always static, you need to check it anyway to throw an exception if appropriate. For other kernels I guess there is should be check that there are only 2 inputs

Reply:
The reason why I ignore the first input is that it doesn't participate in output shape evaluation. It can also be both static and dynamic, I don't see any limitations on it.
I've also added a check on two inputs for other operations, thanks!